### ISSUE TITLE
Serial port

### ISSUE THREAD

**Issue by Lizethgm** [stm32duino/STM32LowPower#85]
I have been trying to test the low power along with the serial ports.
I am using the Generic STM32WL series board "Generic WLE5CCUx" 
My program is this:

```C++
#include "STM32LowPower.h"

String command = "Hello";

void setup() {
  Serial.begin(2400);
  Serial1.begin(2400);
  LowPower.begin();
}

void loop() {
  readMeter();
  delay(500);
  LowPower.deepSleep(10 * 1000);
}

void readMeter()
{  
  Serial1.print(command);
  while(Serial1.available()> 0) 
  {
    Serial.println(Serial1.read());
  }
}

```
I was trying to test it with this code but it only sends the "Hello" command the second time and that's it. If I remove the low power it works correctly. 
How can I make the command "Hello" that I set continue to be sent?

**Comment by fpistm** [comment]
Hi @Lizethgm 
Where `Serial1` is defined or how?

**Comment by fpistm** [comment]
Hi @Lizethgm 
I've made a fix in #87.
I've tested on a WB. I will test WL tomorrow.
